jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/spring/component/JvmStateResolverWorker.java

```diff
@@ -34,7 +34,8 @@
 public class JvmStateResolverWorker {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(JvmStateResolverWorker.class);
-    private static final String STOPPED = "STOPPED";
+    private static final String WINDOWS_SVC_STOPPED = "STOPPED";
+    private static final String LINUX_SVC_STOPPED = "is stopped";
```

Comment:
We control the status string in our Linux shell scripts(jvm-service.sh), let's use "STOPPED" so we don't have to make this distinction.